src/Files.App/UserControls/PathBreadcrumb.xaml.cs

```diff
@@ -1,27 +1,18 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using Files.App.ViewModels;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Input;
+using Files.App.Attributes;
 
 // The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
 
 namespace Files.App.UserControls
 {
+	[DependencyProperty<ToolbarViewModel>("ViewModel")]
 	public sealed partial class PathBreadcrumb : UserControl
 	{
-		public ToolbarViewModel ViewModel
```

Comment:
This seems to be causing an issue when compiling the app
![image](https://github.com/files-community/Files/assets/39923744/15090898-7191-4b40-9405-a78f0b185035)

Reply:
How is that possible? On my side, always MSBuild processed compilation successfully, no matter when.

Reply:
https://dev.azure.com/filescommunity/Files/_build/results?buildId=19041&view=results

Reply:
I think we're missing the `Preview` configuration.

Reply:
@Poker-sang can you take a look at this?

Reply:
Maybe try running for another time? I thought it was an accident, because only updating Package.appxmanifest causes the build failure is unreasonable. And when you revert the revert it was successful again.

Reply:
It works fine on most configs, it only seems to be an issue with the "preview" config.

Reply:
It was successful in  https://github.com/files-community/Files/commit/697f1e14f08394f0dbfdb15aa20b421590934975. It looks like it could run when in preview config

Reply:
And I found that I use Debug config in source generator, when it is Store and Preview config (see Files.sln). Maybe I should fix this after solving this conversation.